bin/disco_creds.py

```diff
@@ -19,6 +19,8 @@ def get_parser():
     parser = argparse.ArgumentParser(description='Disco Credentialaterator')
     parser.add_argument('--debug', dest='debug', action='store_const',
                         const=True, default=False, help='Log in debug level.')
+    parser.add_argument('--silent', dest='silent', action='store_const',
```

Comment:
We should make `--debug` and `--silent` mutually exclusive. See the [docs](https://docs.python.org/2/library/argparse.html#mutual-exclusion).